src/components/SettlementsList.js

```diff
@@ -72,8 +71,17 @@ function SettlementsList(props) {
   );
 }
 
+SettlementsListList.propTypes = {
+  fspNamesById: PropTypes.objectOf({ id: PropTypes.string }).isRequired,
```

Comment:
Is `fspNamesById` used?

Reply:
(And if not, I'm surprised linting didn't catch that)

Reply:
Yes, it's used.